ui/pages/unlock-page/README.mdx

```diff
@@ -12,15 +12,16 @@ Portal page for user to auth the access of their account
 
 ## Props
 
-{/*
+{/\*
 ArgsTable doesn't work with SectionShape
+
 <ArgsTable of={MetaMaskTranslation} />
 */}
```

Comment:
@georgewrmarshall Prettier formatting problem, not totally sure if this is supposed to be visible or not

This is actually a critical error because it prevents Storybook from building 🤷

Reply:
We can remove this! Updated https://github.com/MetaMask/metamask-extension/pull/30440/commits/378cdd4c12ba20828097b645f08d663117c3fba3